docs/version_control.md

```diff
@@ -32,8 +32,6 @@ We encourage using [Commitizen](http://commitizen.github.io/cz-cli/), a great to
   
   
 ```bash
-# clone project 
-git clone https://github.com/brown-ccv/honeycomb.git
```

Comment:
Why delete this?

Reply:
I added the cloning repo step in a quick start, it might be a bit confusing if it was mentioned twice

Reply:
Ah okay gotcha, yeah probably for the best then!